### ISSUE TITLE
suggestions: custom view with timepicker

### ISSUE THREAD

**Issue by noahehall** [intljusticemission/react-big-calendar#1546]
TLDR - remove View.Title and Toolbar and provide a single Header thingy

- I want a single 'calendar view', with a `timespan picker` able to switch tween month, days, whatever tf, and a combo box to select the time period (e.g. beginning + ending things).  the idea being the custom view is smart enough to know if a month view is appropriate, vs a day or week view

- how would you architect (i.e. general big-calendar API) this with the calendar API without touching the source code? (and dont say use the toolbar)
- i'm only interested in removing both the toolbar and the title area, having a single (very minimal) timepicker to adjust the calendar view which will update other business related components elseware on the screen 

*at first...*
I started to create a *regular* custom view, but it requires a class static property `CustomView.title`, which is where I want to render my awesome timepicker thingy

*my solution (cough hack)* ended up chewing [chewing through the codebase](https://github.com/intljusticemission/react-big-calendar/compare/master...noahehall:master) to remove the static class props from custom views and enable sending everything as props (navigate, title, range, etc)

this solution is indeed working, so likely i'll continue with this albeit with the divergence from the big-calendar API (which is what im hoping to avoid)

![super-hacky](https://user-images.githubusercontent.com/10324554/70495277-b4b2d400-1adb-11ea-9812-1691f9860441.gif)
